pkg/apis/policies/v1/policy_test_utils.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
we should rename this to something with the `_test` suffix or these functions will be included in the final binary. (e.g. `utils_test.go`)